Geometry/TrackerCommonData/data/PhaseI/pixbarmaterial.xml

```diff
@@ -131,6 +131,12 @@
       <rMaterial name="materials:CFK"/>
     </MaterialFraction>
   </CompositeMaterial>
+
+  <CompositeMaterial name="InterFlangesSuppTub_CFK" density="1.575*g/cm3" symbol=" " method="mixture by weight">

```

Comment:
@ghugo83 - this new material is identical to the one above. Does it make sense to create an extra one?
@civanch - FYI

Reply:
Hi @ianna 
Thanks for your feedback.

Here I followed the convention which was already there in pixbar.xml, ie that apart from materials:Air, all materials are defined in pixbarmaterial.xml, and trackermaterial.xml.

Then I followed an existing convention in Phase 1 apparently (see pixbarmaterarial.xml for example): the materials are defined separately when it is not obvious, from a Mechanical point of view, that they have same composition.
In my understanding, it is made like that, so that if later on, the density / composition is different, it is trivial to update it in pixbarmaterial.xml

For example here, InterFlangesSuppTub (tubes linking the 2 flanges) do not compulsory have the same CFK as the CFK used for the flange disks.
It is then trivial to update.
In that file, evth follows that logic, for example Flange1_CFK, Flange2_CFK, Flange3_CFK are actually identical, and could have been InnerRadiusFlange_CFK.

Reply:
Of course, not - existing material should be reused.

Reply:
Ok, but there are a lot of materials following this convention in this file. 
Anyway I removed it, because this is a non v1 MB file and now the LogicalSection is defined in v1, so was not worth the hassle in any case.

Reply:
@ghugo83 , I reply without reading of your reply. To be more precise: if it is assumed that density of similar materials may be different we can create different materials for different flanges. This should be a requirement from tracker group, which plan making material budget studies and tuning. In most cases, nobody will change geometry and we should maximally reuse existing material list.